src/js/components/RangeInput/stories/Multi.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import { Box, Button, Grommet, RangeInput } from 'grommet';
+
+import { Add, Subtract } from 'grommet-icons';
+
+const rangeInputTheme = {
```

Comment:
Wondering if we can keep the story entirely focused on the "multi-color track" aspect by removing this custom theming?

Reply:
Sure, will do